assets/scss/7-layout/_container.scss

```diff
@@ -0,0 +1,28 @@
+// Container (l-container)
+//
+// Max width container, centered with margin auto. Note: Demo is best viewed in [standalone preview](/pages/layout/l-container/raw-preview.html).
+//
+// l-container--s - Maxed at 520px
+// l-container--m - Maxed at 650px
+// l-container--l - Maxed at 845px
+// l-container--xl - Maxed at 1080px
+// l-container--xxl - Maxed at 1300px
+//
+// Markup: 7-layout/container.html
+//
+// Styleguide 7.0.1
+//
+.l-container {
+  max-width: 960px;
+  margin: 0 auto;
```

Comment:
@ashley-hebler Tiny little bit of additional feedback here: Can we kill the `margin: 0 auto` in favor of `margin-left: auto` and `margin-right: auto`? Just in case we do indeed want to add some space above and below a container.

Reply:
👍 Can do on the explicit margins.

What's your thinking on REMs as widths? Or would it make more sense if I removed `960px` and used [`$bp-default`](https://github.com/texastribune/ds-toolbox/blob/master/assets/scss/1-settings/_measures.scss#L24)

Reply:
I'm in favor of using REMs as values preeeeetty much everywhere (feel free to talk me out of that). I just think that if we can have dimensions that thoughtlessly scale as text size is adjusted -- both by users and us when we drop the size on the `html` selector -- why wouldn't we?

Reply:
So I tried this out a bit with our plan of setting 62.5% font-size on the body. Sass-mq, for some reason wants `em`-based media queries as opposed to `rem`so I'm still working thorough it. You can mess with your browser prefs font size setting and see how it'd respond [here](https://ttds-preview-flex.surge.sh/pages/layout/l-container/raw-preview.html). Being able to offer users this fluidity would be a pretty nice little enhancement.

Could we lump REMifying our base font-size and containers into our more broad grid discussion? I can get something on our calendar either next week or the following. It's going to take me some time to sync up all these after N4S. I also want to meet with Ben at some point to re-visit our type scale and maybe pull him into our grid discussion.

Reply:
Yes, no rush on that at all! Doesn't even need to be next week ... really whenever you want.

I believe `rem`-based media queries are actually invalid CSS -- only `em`s work. But they'll be identical the same thing because media-query widths will always be based off the size of a root element anyway.